### ISSUE TITLE
Time for a new release?

### ISSUE THREAD

**Issue by hugoboos** [vagrant-libvirt/vagrant-libvirt#837]
The latest release `0.0.40` is from 29th of April this year. Time for a new release?

**Comment by infernix** [comment]
Trying to get closure on https://github.com/fog/fog-libvirt/pull/29, will release soonish though.

**Comment by hugoboos** [comment]
Thanks.

**Comment by dennisdegreef** [comment]
Any news on this? :)

**Comment by infernix** [comment]
Will try to get closure on the fog issue; if no progress in the next few days we'll push a new release

**Comment by nickbroon** [comment]
I agree it's probably time to push a release.

**Comment by pronix** [comment]
https://rubygems.org/gems/vagrant-libvirt/versions/0.0.41

**Comment by nickbroon** [comment]
Did something break?

```
18:49 $ rm -rf ~/.vagrant.d/

18:49 $ vagrant --version
Vagrant 2.0.2

18:50 $ vagrant plugin install vagrant-libvirt
Installing the 'vagrant-libvirt' plugin. This can take a few minutes...
Fetching: excon-0.60.0.gem (100%)
Fetching: formatador-0.2.5.gem (100%)
Fetching: fog-core-1.43.0.gem (100%)
Fetching: fog-json-1.0.2.gem (100%)
Fetching: mini_portile2-2.3.0.gem (100%)
Fetching: nokogiri-1.8.2.gem (100%)
Building native extensions.  This could take a while...
Fetching: fog-xml-0.1.3.gem (100%)
Fetching: ruby-libvirt-0.7.0.gem (100%)
Building native extensions.  This could take a while...
Fetching: fog-libvirt-0.4.2.gem (100%)
Fetching: vagrant-libvirt-0.0.41.gem (100%)
Installed the plugin 'vagrant-libvirt (0.0.41)'!

18:51 $ vagrant up --provider=libvirt ntopng
The provider 'libvirt' could not be found, but was requested to
back the machine 'ntopng'. Please use a provider that exists.
```

**Comment by infernix** [comment]
Yes, fixed in .42

**Comment by nickbroon** [comment]
Sorry for the breakage, but an urgent bug fix/release needed: #862
